examples/chart/teleport-cluster/templates/auth/_config.common.tpl

```diff
@@ -0,0 +1,64 @@
+{{- define "teleport-cluster.auth.config.common" -}}
+{{- $authentication := mustMergeOverwrite .Values.authentication (default dict .Values.authenticationSecondFactor) -}}
+{{- $logLevel := (coalesce .Values.logLevel .Values.log.level "INFO") -}}
+version: v3
+teleport:
+  log:
+    severity: {{ $logLevel }}
+    output: {{ .Values.log.output }}
+    format:
+      output: {{ .Values.log.format }}
+      extra_fields: {{ .Values.log.extraFields | toJson }}
+kubernetes_service:
```

Comment:
I let the kubernetes service bundled in the auth for now, but this could be moved out of the deployment.

Reply:
As far as I understand, proxies are made to be scaled up; I don't think the Kubernetes service needs to. So I tried to change the rolebindings as few as possible. This is also why it stayed in the auth. If we were to move it, I would prefer to move it to a 3rd deployment, maybe even embed `teleport-kube-agent` or something.

Reply:
>I don't think the Kubernetes service needs to

That was my misconception then.
I thought that Kubernetes Service was meant to provide access to K8s cluster, so the more the merrier (to provide HA)